src/lib/flexbox/api/base.ts

```diff
@@ -127,7 +135,7 @@ export abstract class BaseFxDirective implements OnDestroy, OnChanges {
     let value = this._lookupStyle(element, 'display');
 
     // Note: 'inline' is the default of all elements, unless UA stylesheet overrides
-    return value ? value.trim() : 'inline';
+    return value ? value.trim() : ((element.nodeType === 1) ? 'block' : 'inline');
```

Comment:
I'm not sure that checking that the `nodeType === 1` is a valid way to verify that the element is block-level. 1 only means that it's an `Element` node which is true for almost everything except cases like comments or raw text nodes.